content/swiftui/concepts/views/terms/rectangle/rectangle.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'Rectangle'
+Description: 'A rectangular shape aligned inside the frame of the view containing it.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'Rectangle'
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`Rectangle`** view is a rectangular shape that is aligned inside the frame of the `View` containing it.
```

Comment:
```suggestion
The **`Rectangle`** view creates a rectangular shape that is aligned inside the frame of the `View` containing it.
```

Reply:
Upon further thought, maybe this works better

Reply:
In addition, can we re-phrase this sentence?

```suggestion
The **`Rectangle`** view is a pre-built rectangular shape that is aligned inside the frame of the `View` containing it, and can be visually modified as desired.
```

Reply:
I changed it ! thank you ~!